db/migrate/20250418004442_change_to_polymorphic_association_in_teams.rb

```
@@ -3,9 +3,13 @@
 class ChangeToPolymorphicAssociationInTeams < ActiveRecord::Migration[8.0]
   def change
     # Remove old assignment reference (course reference doesn't exist)
-    remove_reference :teams, :assignment, foreign_key: true
+    if column_exists?(:teams, :assignment_id)
```

RuboCop (1.86.1)

[convention] Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

(Style/IfUnlessModifier)